doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -235,7 +235,6 @@ Please note that "shared" is a reference to multi-tenant support and pooled stor
 | `req.kcm.01` | General | The Architecture **must** allow infrastructure resource sharing within a Kubernetes cluster. |
 | `req.kcm.02` | General | The Architecture **must** support discoverability of nodes and their features. |
 | `req.kcm.03` | General | The Architecture **must** support scheduling of workloads based on Enhanced Platform Awareness (EPA) features such as CPU Pinning, huge-pages and SR-IOV. |
-| `req.kcm.04` | General | The Architecture **must** include kubernetes artefacts (e.g., images, Helm charts, etc.) repository capabilities. |
```

Comment:
I'm not sure why this line has been deleted?

Reply:
You're right didn't mean to delete this. Did ctr-x not ctr-c. On a related note, should this also be moved as a requirement under API. Something like "The Architecture **must** support the usage of an OCI compatible artefact repository.

Reply:
Yes agreed- could you raise an Issue for that please?

Reply:
And could you add this back in please?

Reply:
added in #1108 and will add back in